### PR TITLE
Delete map fix #78

### LINKED ISSUE #78: Delete a Domain

### PR DESCRIPTION
**Pull request by rkalra247**

### CODE REVIEW

File: src/public/client.ts [hunk 1]
```diff
@@ -55,6 +55,42 @@ fetch('/api/mappings')
...
   })
+const render = () => {
+  fetch('/api/mappings')
+    .then(r => r.json())
+    .then((data: Array<Mapping>) => {
+      data.reverse()
+      domainList.innerHTML = ''
+      data.forEach(e => {
+        if (e.domain && e.port) {
+          const eachDomain = document.createElement('div')
```
Comment (songz) [inline src/public/client.ts]: look at what you did in line 54. `new DisplayMap(e)`
Reuse the classes you have already built.

File: src/public/client.ts [hunk 2]
```diff
@@ -55,6 +55,42 @@ fetch('/api/mappings')
...
+          eachDomain.innerHTML = `
+        <li class="list-group-item" style="display: flex;">
+        <a href="">${e.domain}</a>
+        <small class="form-text text-muted" style="display: inline-block;">PORT: ${e.port}</small>
+        <hr />
+        <hr/>
+        <div class="deleteButton" href="/">Delete</div>
+        </li>
+      `
+          const delButton = helper.getElement('.deleteButton', eachDomain)
```
Comment (songz) [inline src/public/client.ts]: move into `DesplayMap` class.

File: src/public/client.ts [hunk 3]
```diff
@@ -76,6 +112,11 @@ create.onclick = (): void => {
...
   }).then(() => {
-    window.location.reload()
+    render()
   })
+  port.value = ''
+  ipAddress.value = ''
+  subDomain.value = ''
 }
+
+render()
```
Comment (songz) [inline src/public/client.ts]: if you are calling render, then you can remove your `fetch('/api/mappings'...)` that you did outside of `render`

File: src/public/client.ts [hunk 4]
```diff
@@ -55,6 +55,42 @@ fetch('/api/mappings')
...
+        <li class="list-group-item" style="display: flex;">
+        <a href="">${e.domain}</a>
+        <small class="form-text text-muted" style="display: inline-block;">PORT: ${e.port}</small>
+        <hr />
+        <hr/>
+        <div class="deleteButton" href="/">Delete</div>
+        </li>
+      `
+          const delButton = helper.getElement('.deleteButton', eachDomain)
+          console.log('deleteButton exists')
```
Comment (allopez7) [inline src/public/client.ts]: delete this

File: src/public/client.ts [hunk 5]
```diff
@@ -43,12 +43,25 @@ class DisplayMap {
         <div class="deleteButton" href="/">Delete</div>
     </li>
 `
+
+      const delButton = helper.getElement('.deleteButton', mappingElement)
+      console.log('deleteButton exists')
```
Comment (songz) [inline src/public/client.ts]: remove `console.log`

**Pull request merged**